src/instrumenter/instrumentation_context.ts

```diff
@@ -72,6 +72,7 @@ export class InstrumentationContext {
     public readonly outOfContractFlagName: string;
     public readonly scratchField: string;
     public readonly checkInvsFlag: string;
+    public readonly encodedLoggerArgs: string | null;
```

Comment:
I think we can remove this field altoghether. (See coments on getDebugInfo in instrument.ts.